src/Interpreters/InternalTextLogsQueue.cpp

```diff
@@ -46,11 +46,11 @@ void InternalTextLogsQueue::pushBlock(Block && log_block)
         LOG_WARNING(&Poco::Logger::get("InternalTextLogsQueue"), "Log block have different structure");
 }
 
-const char * InternalTextLogsQueue::getPriorityName(int priority)
+std::string_view InternalTextLogsQueue::getPriorityName(int priority)
 {
     /// See Poco::Message::Priority
-
-    static constexpr const char * const PRIORITIES[] =
+    /// NOTE: std::make_array() is experimental now, hence we need to pass the size.
+    static constexpr const std::array<std::string_view, 10> PRIORITIES =
```

Comment:
```suggestion
    using namespace std::literals;
    static constexpr std::array PRIORITIES =
    {
        "Unknown"sv,
        ...
     }
```
Should work to avoid using size